### ISSUE TITLE
WARN  Cannot stringify a function isRel and getRelated error

### ISSUE THREAD

**Issue by alexd73** [mrichar1/jsonapi-vuex#73]
Hi! If i use an action `get`, I get this error in console

>  WARN  Cannot stringify a function isRel
WARN  Cannot stringify a function isAttr
<details>
  <summary>this returned from jv </summary>

```
{
  "_jv": {
    "type": "node--page",
    "id": "d9e71e78-f50e-46f2-87b6-c7ebfbd37c08",
    "relationships": {
      "node_type": {
        "data": {
          "type": "node_type--node_type",
          "id": "e48ba823-9d21-4c9e-beab-69d048908d70"
        },
        "links": {
          "self": {
            "href": "https://localhost/api/node/page/d9e71e78-f50e-46f2-87b6-c7ebfbd37c08/relationships/node_type?resourceVersion=id%3A29"
          },
          "related": {
            "href": "https://localhost/api/node/page/d9e71e78-f50e-46f2-87b6-c7ebfbd37c08/node_type?resourceVersion=id%3A29"
          }
        }
      },
      "revision_uid": {
        "data": {
          "type": "user--user",
          "id": "ff9767ae-fe0b-4bdf-ade6-514d92cac680"
        },
        "links": {
          "self": {
            "href": "https://localhost/api/node/page/d9e71e78-f50e-46f2-87b6-c7ebfbd37c08/relationships/revision_uid?resourceVersion=id%3A29"
          },
          "related": {
            "href": "https://localhost/api/node/page/d9e71e78-f50e-46f2-87b6-c7ebfbd37c08/revision_uid?resourceVersion=id%3A29"
          }
        }
      },
      "uid": {
        "data": {
          "type": "user--user",
          "id": "ff9767ae-fe0b-4bdf-ade6-514d92cac680"
        },
        "links": {
          "self": {
            "href": "https://localhost/api/node/page/d9e71e78-f50e-46f2-87b6-c7ebfbd37c08/relationships/uid?resourceVersion=id%3A29"
          },
          "related": {
            "href": "https://localhost/api/node/page/d9e71e78-f50e-46f2-87b6-c7ebfbd37c08/uid?resourceVersion=id%3A29"
          }
        }
      },
      "sections": {
        "data": [
          {
            "type": "paragraph--items",
            "id": "e6b9e27c-0bfd-4f64-8e40-086e6d811a2e",
            "meta": {
              "target_revision_id": 89
            }
          },
          {
            "type": "paragraph--items",
            "id": "e06a503b-b30d-46cd-aa4c-ba606fa0405c",
            "meta": {
              "target_revision_id": 93
            }
          }
        ],
        "links": {
          "self": {
            "href": "https://localhost/api/node/page/d9e71e78-f50e-46f2-87b6-c7ebfbd37c08/relationships/sections?resourceVersion=id%3A29"
          },
          "related": {
            "href": "https://localhost/api/node/page/d9e71e78-f50e-46f2-87b6-c7ebfbd37c08/sections?resourceVersion=id%3A29"
          }
        }
      }
    },
    "links": {
      "self": {
        "href": "https://localhost/api/node/page/d9e71e78-f50e-46f2-87b6-c7ebfbd37c08?resourceVersion=id%3A29"
      }
    },
    "isRel": {},
    "isAttr": {}
  },
  "drupal_internal__nid": 1,
  "drupal_internal__vid": 29,
  "langcode": "ru",
  "revision_timestamp": "2019-06-11T12:28:09+00:00",
  "revision_log": null,
  "status": true,
  "title": "Ваша кредитная система",
  "created": "2019-05-31T15:50:33+00:00",
  "changed": "2019-06-11T12:28:09+00:00",
  "promote": false,
  "sticky": false,
  "default_langcode": true,
  "revision_translation_affected": null,
  "metatag": null,
  "path": null,
  "body": null,
  "subtitle": {
    "value": "Title",
    "format": "basic_html",
    "processed": "Title"
  },
  "node_type": {},
  "revision_uid": {},
  "uid": {},
  "sections": {}
}
```
</details>

And calling an action `getRelated` I get error in browser

> Cannot use 'in' operator to search for 'data' in undefined
<details>
  <summary>I use this JSON format </summary>

```
{
  jsonapi: {
    version: "1.0",
    meta: {
      links: {
      self: {
        href: "http://jsonapi.org/format/1.0/"
      }
    }
  }
},
  "data": [
    {
      "type": "node--page",
      "id": "string",
      "attributes": {
        "drupal_internal__nid": 0,
        "drupal_internal__vid": 0,
        "langcode": {
          "value": "string",
          "language": null
        },
        "revision_timestamp": 0,
        "revision_log": "",
        "status": true,
        "title": "string",
        "created": 0,
        "changed": 0,
        "promote": false,
        "sticky": false,
        "default_langcode": true,
        "revision_default": true,
        "revision_translation_affected": true,
        "metatag": {},
        "path": {
          "alias": "string",
          "pid": 0,
          "langcode": "string",
          "pathauto": 0
        },
        "body": {
          "value": "string",
          "format": "string",
          "summary": "string"
        },
        "subtitle": {
          "value": "string",
          "format": "string"
        }
      },
      "relationships": {
        "node_type": {
          "data": {
            "type": "node_type--node_type",
            "id": "string"
          }
        },
        "revision_uid": {
          "data": {
            "type": "user--user",
            "id": "string"
          }
        },
        "uid": {
          "data": {
            "type": "user--user",
            "id": "string"
          }
        },
        "menu_link": {
          "data": {
            "type": "menu_link_content--menu_link_content",
            "id": "string"
          }
        },
        "sections": {
          "data": [
            {
              "type": "string",
              "id": "string"
            }
          ]
        }
      },
      "links": {
        "property1": {
          "href": "string",
          "meta": {}
        },
        "property2": {
          "href": "string",
          "meta": {}
        }
      },
      "meta": {}
    }
  ],
  "meta": {},
  "links": {
    "property1": {
      "href": "string",
      "meta": {}
    },
    "property2": {
      "href": "string",
      "meta": {}
    }
  },
  "jsonapi": {
    "version": "string",
    "meta": {}
  }
}
```
</details>

It wrong JSON:API format? Help me understand this mistake )
Thank you!

**Comment by mrichar1** [comment]
Hi,
> WARN Cannot stringify a function isRel
This is an error from the `JSON.stringify()` method, which doesn't know how to turn the `isRel` and `isAttr` functions into JSON. I assume you are calling this on the results of the `get` action to review the normalised records.

Since these functions make no sense when translated into JSON, it's entirely correct that this warning should appear, and that these functions don't appear in the output.

> Cannot use 'in' operator to search for 'data' in undefined

Can you add the code you are using to call the action and process the resulting data? It's not clear if the `data` mentioned here is something in the module code, or in the calling code.

**Comment by alexd73** [comment]
> functions into JSON. I assume you are calling this on the results of the get action to review the normalised records (or vue is calling it indirectly when using something like {{data }}).

I call `get` from action like this:

```
async fetchFrontFAQ({ dispatch, commit }) {
  const params = { 'filter[promote][value]': 1 }
  const items = await dispatch('jv/get', ['node/faq', { params: params }], {
    root: true
  })
  commit('setFrontFAQ', items)
}
```

And I have this warnings by calling only `await store.dispatch('FAQ/fetchFrontFAQ')` (without any other manipulations)
> Can you add the code you are using to call the action and process the resulting data? 

Yes! It is my code:
```
    const paragraph = await dispatch(
      'jv/get',
      ['node/page/d9e71e78-f50e-46f2-87b6-c7ebfbd37c08', { params: params }],
      { root: true }
    )

```
Will it help you understand what I'm doing wrong?

**Comment by mrichar1** [comment]
**Probably** fixed by #74 - closing issue as reporter is now not using this library.
